src/oc-id/app/controllers/profiles_controller.rb

```diff
@@ -21,10 +21,13 @@ def update
     # Note that if an email address is provided, don't update it right away.
     # Instead send a confirmation email and let that email link update the
     # email address.
-    #update note
-    params[:user].reject! { |k| k == :email }
+    if user_params.key?(:email) && @user.email != user_params[:email]
```

Comment:
https://github.com/chef/chef-server/pull/2329 is the source for these changes. Let us not revert it.